tests/components/samsungtv/snapshots/test_init.ambr

```diff
@@ -30,8 +30,10 @@
       }),
       'manufacturer': None,
       'model': '82GXARRS',
+      'model_id': None,
       'name': 'fake',
       'name_by_user': None,
+      'primary_config_entry': <ANY>,
```

Comment:
I am honestly wondering why this test didn't fail before

Reply:
Me too

Reply:
The test itself also looks normal without any reason why it would be different